lib/customers.js

```diff
@@ -569,6 +570,77 @@ function updatePhotoCard (id, patch) {
     })
 }
 
+/**
+ * @param {String} imageData image encoded
+ * @param {String} directory directory path of id card data for a certain user
+ */
+
```

Comment:
An extra newline here.

Normally, I would say to remove the comments, but as it is a function which touches on lower level stuff, I like the extra comments